prosody/Dockerfile

```diff
@@ -2,11 +2,19 @@ FROM jitsi/base
 
 RUN \
 	apt-dpkg-wrap apt-get update && \
-	apt-dpkg-wrap apt-get install -t stretch-backports -y prosody && \
+	apt-dpkg-wrap apt-get install -t stretch-backports -y prosody luarocks gcc libssl1.1 libssl1.0-dev git && \
```

Comment:
do we want to install these new packages from backports too? why?

Reply:
Yeah I think we don't need to do that. I will fix it in my next update.